test/FunctionMetadataGeneratorTests/SdkTests.csproj

```diff
@@ -25,6 +25,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.25" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Script.Abstractions" Version="1.0.0-preview" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.11.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="3.11.0" />
```

Comment:
I had to add these package reference entries with specific version to resolve the version conflict issue.

`SdkTests` depends on `FunctionsMetadataLoaderExtension` which depends on `WebJobs.Script.Abstractions 1.0.0-preview` which pulls in an older version of `Microsoft.CodeAnalysis.CSharp.Workspaces`. This caused the version conflict (with the more recent version of `Microsoft.CodeAnalysis.CSharp.Workspaces` I used in the Analyzer project. (This is a new dependency to analyzer project as we did not have any code fix providers so far).

![image](https://user-images.githubusercontent.com/144469/132589691-ed128575-d4e8-4308-bc4f-0438367f3755.png)